components/gly_gen/glycosight/sup_functions.ts

```diff
@@ -0,0 +1,21 @@
+import { GlycoSightOutputType, GlycoSightFileURLType } from "./data_models";
+import { GlycoSightOutputNode } from ".";
+import { z } from "zod";
+
+const devGSURL = "http://localhost:5000/api/"
```

Comment:
I'm assuming this is your API? do is this still being used? maybe it needs to be an environment variable so we can update it (?)

Reply:
This 100% needs to be an environment variable. (In production, it should point to `https://aws.glygen.org/glycosight/api`)

Reply:
process.env.GLYGEN_GLYCOSIGHT_API perhaps
add it to `.env.example`

Reply:
This has been updated to inspect NODE_DEV for "development", and defaults to our API endpoint if that isn't available. I don't have a resolvable DNS for development between our API and a testing DRS target, but this address should work out of the box.

Values from `.env.example` isn't being picked up in my stock environment, and `.env` is protected by `.gitignore` so I'm hesitant to touch that.

Reply:
Hi @pmcneely -- .env.example is copied to .env if it doesn't exist. It's just about documenting that it is an environment variable that can/should be configured.

Reply:
@u8sand I see. It looks like the file is not removed automatically after shutting down the dev environment, masking updates to `.env.example`. Our API is only either dev or production, so the code as is should work.